vms/platformvm/warp/payload/addressed_payload.go

```diff
@@ -0,0 +1,56 @@
+// Copyright (C) 2019-2023, Ava Labs, Inc. All rights reserved.
+// See the file LICENSE for licensing terms.
+
+package payload
+
+import "fmt"
+
+// AddressedPayload defines the format for delivering a point to point message across VMs
+// ie. (ChainA, AddressA) -> (ChainB, AddressB)
+type AddressedPayload struct {
+	SourceAddress []byte `serialize:"true"`
```

Comment:
I guess we want to make this generic for all possible chains? (EVM chains use 20 bytes addresses)

Reply:
Yup, the intent here is to generalize this so that other VMs can use the same format. This will require changing the unpacking in Subnet-EVM / Teleporter as well.